src/main/java/movies/spring/data/neo4j/movies/PersonInMovie.java

```diff
@@ -10,14 +10,14 @@
  * @author Michael J. Simons
  */
 @RelationshipProperties
-public class Actor {
+public class PersonInMovie {
```

Comment:
renamed because there are also directors and producers (not sure what the umbrella word is, in that case)